### ISSUE TITLE
Selected State does not annonce when Picker Component selected

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30957]
## Description
Does not announce "selected" on the focus of Picker Component.

This issue is covered in-depth by parent issue #30842, please check there for more details.

## React Native version:
0.63

**Comment by fabriziobertoglio1987** [comment]
as the picker component is deprecated https://reactnative.dev/docs/picker
I believe this issue should be opened in the relevant community packages https://reactnative.directory/?search=picker :peace_symbol: :pray:

**Comment by amarlette** [comment]
Closing this issue because the component has been deprecated.

Issue opened with respective repository react-native-picker/picker#215
